config/deploy_development.env

```diff
@@ -6,8 +6,8 @@ CARD_CREATOR_PASSWORD=AQECAHjqALewp8JBJNxIQvR4oY795dyG7INaGR1glMsTEgetggAAAJ4wgZ
 
 #      Encrypted Values for API v0.2
 # ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----
-ILS_CLIENT_KEY=AQECAHjqALewp8JBJNxIQvR4oY795dyG7INaGR1glMsTEgetggAAAHoweAYJKoZIhvcNAQcGoGswaQIBADBkBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDJPuxivxfMCssbQcQAIBEIA3dgSMfy540wG69KJECgAiMq6DVVi0bPJwi7nXJ8LI/I37MMGzONmQjapdWJv17ceMzSm5gEeCBA==
-ILS_CLIENT_SECRET=AQECAHjqALewp8JBJNxIQvR4oY795dyG7INaGR1glMsTEgetggAAAIYwgYMGCSqGSIb3DQEHBqB2MHQCAQAwbwYJKoZIhvcNAQcBMB4GCWCGSAFlAwQBLjARBAzPb80eDfdT9AMQ9ucCARCAQlxE/9PXuDhq0teK1IdZQxjmcf9bW2mfxEdT38yrPfnxk4mRCYwl9HL9iug9dnzH3KpwumsFFgrflcVQk1Hd1m7jEA==
+ILS_CLIENT_KEY=AQECAHjqALewp8JBJNxIQvR4oY795dyG7INaGR1glMsTEgetggAAAHoweAYJKoZIhvcNAQcGoGswaQIBADBkBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDJSW/4wv/vPtqNFgbgIBEIA3bFZJLvQqiqhLkRyo2bkFo7yly2cNTD+H831fwZKzBHSEgLwI3VcJz+5HFdly1EppqXZIgjX2NA==
```

Comment:
These are encrypted values which had to be updated after last week's Sierra upgrade.